tests/model_hub_tests/torch_tests/test_transformers.py

```diff
@@ -0,0 +1,286 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
please pay attention to that all models tests results will be incorporated into the single html report. Consider to split them by separate run in GA config file

Reply:
Single report is fine, don't see a reason to separate